packages/block-editor/src/components/inserter/style.scss

```diff
@@ -89,6 +89,9 @@ $block-inserter-search-height: 38px;
 }
 
 .block-editor-inserter__results {
+	align-items: center;
+	justify-content: center;
+	display: flex;
```

Comment:
This looks like where the centering problem exists. It's effecting all items in the Inserter.

Reply:
Yes I fix it in the next commit :) Thanks !